docs/pages/reference/resources.mdx

```diff
@@ -145,6 +145,10 @@ are required.*/}
 | <pre>kubernetes_groups: <br/>  - "system:masters" <br/>kubernetes_labels: <br/>  env: ["dev"] <br/>kubernetes_resources: <br/>  - name: "&ast;" <br/>    kind: pods <br/>    namespace: "foo" <br/>  - name: "&ast;" <br/>    kind: deployments <br/>    api_group: apps<br/>    namespace: "foo"</pre> | ⚠️ not supported                                                                                                     | ⚠️ not supported                                                                                                     | ⚠️ not supported                                                                                                                                                                                           | ✅ can exec in pods in `foo` <br/>✅ can access `deployments` in `foo` <br/>❌ cannot exec in other namespaces <br/>❌ cannot access `deployments` in other namespaces <br/>❌ cannot access `configmaps` in `foo`  |
 
 
+SAML IdP role option `spec.idp.saml.enabled: true/false` is only supported in role version 7 and below.
```

Comment:
This paragraph might be difficult to find if users assume that the "Role versions" section is exclusively about `kubernetes_resource` fields (which it is, except for this one paragraph). I would:
- Within the "Role versions" H3, add an H4 for Kubernetes resources and another H4 for the SAML IdP option
- Add a brief introduction to the "Role version" H3 that explains that roles have different versions and this section explains the differences between them.

Reply:
Agree, updated that in https://github.com/gravitational/teleport/pull/55605/commits/05e5998fc746b9a0282761e811e29efcd5d0401e. 

I have added just a minimum description in "Role versions" H3. Do suggest if you find anything that is worth expanding there.